pytorch/inference/docker/1.10/py3/Dockerfile.graviton.e3.cpu

```diff
@@ -103,7 +105,7 @@ RUN conda install -c \
 # install torch and torchvision from the PyTorch website
 RUN pip uninstall -y torch \
  && pip install --no-cache-dir -U $PT_INFERENCE_URL
-RUN pip install torchvision
+RUN pip install torchvision==0.11.1
```

Comment:
Why pin this?

Reply:
nice call, rather not pin if we do not need to, but might be required for torchserve. going to remove the pin and run the tests on it to see if all is good.

Reply:
the pin is required but can be more general. this is updated to stay in 0.11.x